lib/L.js

```diff
@@ -8,8 +8,17 @@
  * - Lm = Letter, modifier
  * - Lo = Letter, other
  *
- * This list has been computed from http://unicode.org/Public/UNIDATA/UnicodeData.txt
+ * This list has been computed from http://www.unicode.org/Public/5.2.0/ucdxml/ucd.all.flat.zip
+ *
+ * 1) Get and unzip unicode database data. Then prettify it so each <char> attribute is on its own
+ * line.
+ *
+ * wget http://www.unicode.org/Public/5.2.0/ucdxml/ucd.all.flat.zip && unzip ucd.all.flat.zip && xmllint --format ucd.all.flat.xml > ucd.all.flat.xml.pretty
+ *
+ * 2) Extract relevant data.
+ *
+ * grep -E '<char cp="[0-9A-F]{4}"' ucd.all.flat.xml.pretty | grep -E 'gc="L[ultmo]"' | perl -n -e '/cp="([0-9A-F]{4})/ && print "$1\n"' | scripts/convert-to-regexp.sh
```

Comment:
I didn't know where to put the instructions. But these are the steps I took to compile the regular expression.